server/storage/interface.go

```diff
@@ -0,0 +1,9 @@
+package storage
+
+type MetaStore interface {
+	UpdateCurrent(gun, role string, version int, data []byte) error
```

Comment:
Eventually we will want functions that operate on old version (OK, maybe there won't be an Update for old versions, but there will be a Get). The ___Current versions are shortcuts to interact with the most recent version and they are relevant now.